styles/_contact.scss

```diff
@@ -49,26 +56,27 @@
     }
   }
 }
-@media only screen and (max-width: 768px) {
+@media only screen and (max-width: 990px) {
+  h2 {
+    font-size: 40px;
+    padding-top: 80px;
+  }
   .contact {
-    h2 {
-      top: 100px;
-      font-size: 40px;
-    }
     form {
-      padding-top: 180px;
+      padding-top: 60px;
```

Comment:
May need to verifiy if it's enough for different screen size. 
Same for the next media query (for screen ->  min-height: 1000px)